### PR TITLE
chore: upgrade to v3.4.0-0 prerelease

### PR DESCRIPTION
**Pull request by nphyatt**

### CODE REVIEW

File: src/app/services/browser-auth/browser-auth.service.ts [hunk 1]
```diff
@@ -14,14 +14,15 @@ export class BrowserAuthService implements IdentityVault {
   constructor(private storage: Storage) {}
 
   config = {
-    authMode: AuthMode.InMemoryOnly,
+    authMode: AuthMode.SecureStorage,
```
Comment (kensodemann) [inline src/app/services/browser-auth/browser-auth.service.ts]: What does `AuthMode.SecureStorage` do in the browser? I guess the answer is that it does whatever the `BrowserAuthService` does, and that is coded by the user, so maybe my question is, why the change here?

Comment (nphyatt) [reply src/app/services/browser-auth/browser-auth.service.ts]: Well in the browser implementation the mode is fairly trivial/non-consequential in all honesty unless the implementor decides to do something with it but I suppose the reason I changed it was that it lined up more with my expectations of how the typical user would implement browser mode. InMemory mode in the plugin means all session data is lost when the app is closed or the vault locked but SecureStorage mode means the session data persists across app close/open and lock is a no-op which is how I would guess most BrowserAuth implementations would work

**Comment by kensodemann** [comment]
LGTM

I _think_ the odd behavior I found and reported in Slack is in IV itself and not the app, so I believe this can be merged (with a couple of obvious mods) once 3.4.0 is out of pre-release.

One item of note, is that the about.html has a hard code IV version in it. That should be updated. I don't know if there is a good way to get it out of IV itself, but if we could that would be ideal.

https://github.com/ionic-team/cs-demo-iv/blob/master/src/app/about/about.page.html#L17

**Comment by kensodemann** [comment]
I have implemented v3.4.3 in the main branch as well as the secure storage stuff out of this PR, so I believe this PR is now outdated.
